tests/js/client/shell/shell-admin-cluster-rebalance.js

```diff
@@ -29,12 +29,18 @@
 let jsunity = require('jsunity');
 const arangodb = require('@arangodb');
 const db = arangodb.db;
-const { getDBServers } = require('@arangodb/test-helper');
+const {getDBServers} = require('@arangodb/test-helper');
+const internal = require('internal');
 const database = "cluster_rebalance_db";
+
+const suspendExternal = internal.suspendExternal;
+const continueExternal = require("internal").continueExternal;
+const wait = require("internal").wait;
 let prevDB = null;
+let dbServer = null;
```

Comment:
Instead of making them global variables, can these be moved inside the `rebalanceSuite`?